src/rm.js

```diff
@@ -48,7 +48,25 @@ function rmdirSyncRecursive(dir, force) {
 
   var result;
   try {
-    result = fs.rmdirSync(dir);
+    // Retry on windows, sometimes it takes a little time before all the files in the directory are gone
+    var start = Date.now();
+    while (true) {
+      try {
+        result = fs.rmdirSync(dir);
+        if (fs.existsSync(dir)) throw { code: "EEXIST" }
```

Comment:
I think this is fine, though since it's potentially user-facing, I'd like to invest a little more time in the thrown Error:

1. Throw an actual `Error`. No base class needed.
2. `EEXIST` is specific to creating files. Could we use a more generic error like `EAGAIN` or even `EOTHER`?
3. A good message.

@arturadib - Do we have a precedent for these kinds of errors?